### PR TITLE
Final Milestone - JavaScript Group Capstone Project - PR

### PR DESCRIPTION
**Pull request by ITurres**
# My Skills JavaScript Capstone - `WebPack` `ES6` `API` `Jest`

## 👋 Hi Dear CR!, It's @Ruthmy and @ITurres here 👋

In this project, we built an API-based web application, and so we created a movie site that uses the [TV Maze API](https://www.tvmaze.com/api) to get data about movies and the [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) provided by Microverse to listen to users' likes and comments. The site allows users to browse movies, view their details, and add likes and comments.

---

#### The goals of this project were to:

- Create a movie site that is easy to use and navigate.
- Provide users with access to information about movies, including their summaries.
- Allow users to interact with the site by adding likes and comments.

---

## 🎯 **Project Goals:**

- [x] Use the Kanban board to divide and separately manage the project requirements as a group of small tasks to achieve the objective.
- [x] Understand how to use medium-fidelity wireframes to create a UI.
- [x] The site must use the `TV Maze API` to fetch data about TV shows.
- [x] The site must use the `Involvement API` to allow users to add likes and comments on shows.
- [x] Follow the [GitFlow as a workflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) to create and unify the different features of the project.
- [x] Follow the best practices for [HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) and [JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).
- [x] Documented the work in a [professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
- [x] ES6 syntax implemented and project divided into modules.
- [x] Configure WebPack 5 with Linters for HTML, CSS, JavaScript, and the JavaScript testing library Jest.
- [x] The project has a home page showing a list of items (shows) that users can "like" and comment on.
- [x] By clicking on the "Comments" button, a pop-up window appears with more information about the show, the existing comments about it, and the form to create a new comment.
- [x] Counters for the number of shows displayed on the screen and for the number of comments each show has.
- [x] [Peer code reviews](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md#code-review) were performed for each of the delivered tasks, writing clear, professional, and respectful review comments for team members.
- [x] Unit tests were performed with Jest to check the performance of the implemented counters.

---

### 🚀 [Live Demo Link](https://iturres.github.io/TVmaze-movies-website/dist/)

---

### README issues/page link

> Once this branch is merged into the `main` branch, the **Link** will redirect to the issues page. However, at the moment, it will result in a 404 error due to the relative path.

#### Webhint:

> As you might already be aware, It appears that the Webhint modules on GitHub Actions have been experiencing ongoing issues for approximately the last three weeks. As a result of these issues, the Webhint module has been removed from the general Linters.

---

### Thank you for taking the time to review this PR ⭐!

---

If you require additional information or have any questions, please feel free to contact me on Slack as _**Arturo (Arthur) Emanuel Guerra Iturres**_. I'll be happy to assist you. 🎯

---

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,251 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+---
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```
Comment (zieeco) [inline README.md]: - Hey team, the MIT link is currently broken, please, kindly fix this to make your readme more professional.

File: src/js/services/involvement-API/getLikes.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import involvementAPI from './involvement-API.js';
+
+// Function to get the likes of each Show  -----------------------------------------------------
+// eslint-disable-next-line consistent-return
```
Comment (zieeco) [inline src/js/services/involvement-API/getLikes.js]: - Please, let's remove this eslint disable as it has no effect on your code also, it's not a good practice disabling the linter as this helps to improve our development skills.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,





Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Good code organization:heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ITurres) [reply README.md]: Hi @zieeco ! How are you? thank you! yes, you are absolutely right, the link is currently not pointing to where it should. We will make make the changes right away 🎯

File: src/js/services/involvement-API/getLikes.js (hunk 2 above)
Comment (ITurres) [reply src/js/services/involvement-API/getLikes.js]: @zieeco, 🙏🏼 thank you to point this out! We have overlooked it, again, we will the changes required right away! Cheers, and happy coding!

**Review by Owusu-Desmond — APPROVED** [review]
Hi @ITurres and @Ruthmy  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ITurres** [comment]
Hi @Owusu-Desmond! Thank you for reviewing this PR 😃! Happy weekend and happy coding!

**Pull request merged**